File: C/BEGINNER/2140 Two Bills.c
#include <stdio.h>

int main()
{
    int value, payment;
    scanf("%d %d", &value, &payment);
    while ((value != 0) || (payment != 0))
    {
        int change;
        change = payment - value;
        for (int i = 0; i < 2; i++)
        {
            if (change >= 100)
                change -= 100;
            else if (change >= 50)
                change -= 50;
            else if (change >= 20)
                change -= 20;
            else if (change >= 10)
                change -= 10;
            else if (change >= 5)
                change -= 5;
            else if (change >= 2)
                change -= 2;
            else
                change -= 1;
        }
        if (change == 0)
            printf("possible\n");
        else
            printf("impossible\n");
        scanf("%d %d", &value, &payment);
    }
    return 0;
}

File: Python/BEGINNER/2140 Two Bills.py
# SUBPROGRAM
# MAIN PROGRAM
value, payment = map(int, input().split())
while ((value != 0) or (payment != 0)):
    change = payment - value
    for i in range(2):
        if change >= 100:
            change -= 100
        elif change >= 50:
            change -= 50
        elif change >= 20:
            change -= 20
        elif change >= 10:
            change -= 10
        elif change >= 5:
            change -= 5
        elif change >= 2:
            change -= 2
        else:
            change = -1
    if change == 0:
        print("possible")
    else:
        print("impossible")
    value, payment = map(int, input().split())